cvmfs/upload_local.h

```diff
@@ -13,6 +13,8 @@
 #include "upload_facility.h"
 #include "util_concurrency.h"
 
+struct Counters;
```

Comment:
This line should not be here.
`counters_` has this type: `UniquePtr<UploadCounters>`
`struct UploadCounters` is defined in `upload_facility.h`